isofit/configs/sections/radiative_transfer_config.py

```diff
@@ -176,11 +176,11 @@ def __init__(self, sub_configdic: dict = None, name: str = None):
         """float: 6s-only view azimuth."""
 
         self._earth_sun_distance_file_type = str
-        self.earth_sun_distance_file = None
+        self.earth_sun_distance_file = ""
         """str: 6s-only earth-to-sun distance file."""
 
         self._irradiance_file_type = str
-        self.irradiance_file = None
+        self.irradiance_file = ""
```

Comment:
@evan-greenbrg Ok, changing this to `""` causes `self.irradiance_file` to never be `Non`e, right?

Reply:
Exactly